cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -265,6 +270,14 @@ private KafkaRebalanceStatus updateStatus(KafkaRebalance kafkaRebalance,
                     throw new IllegalArgumentException("The brokers list is mandatory when using the " + mode.toValue() + " rebalancing mode");
                 }
                 break;
+            case REMOVE_DISKS:
+                rebalanceOptionsBuilder = new RemoveDisksOptions.RemoveDisksOptionsBuilder();
+                if (brokerAndVolumeIds != null && !brokerAndVolumeIds.isEmpty()) {
```

Comment:
So, should this also check the content of the `BrokerAndVolumeIds` objects? I think this validation will let something like this pass:

```yaml
  moveReplicasOffVolumes:
    - brokerId: 0
      volumeIds: []
```

It seems like either we should validate this as well?

Reply:
Yep, I should fix this

Reply:
PS: It does not really replace the check here. But @ppatierno used in his PR `@MinimumItems(1)` to enforce that empty lists will not be allowed. So maybe you should do it as well?

Reply:
Added the minimumItems(1) annotation to volumes and also on the `getMoveReplicasOffVolumes()`